index.html

```diff
@@ -0,0 +1,265 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>The Environment Summit 2021</title>
+  </head>
+  <body>
+    <header class="mobile-nav">
+      <input type="checkbox" class="nav-checkbox" id="nav-switch" />
+      <label for="nav-switch" class="nav-btn">
+        <span class="nav-icon">&nbsp;</span>
+      </label>
+      <div class="nav-bck">&nbsp;</div>
+      <nav class="nav-cont">
+        <ul class="mob-cont-link">
+          <li class="mob-single-link">
+            <a href="./about.html" class="mob-link">About</a>
+          </li>
+          <li class="mob-single-link">
+            <a href="#main" class="mob-link">Program</a>
+          </li>
+          <li class="mob-single-link">
+            <a href="#join" class="mob-link">Join</a>
+          </li>
+          <li class="mob-single-link">
+            <a href="#sponsor" class="mob-link">Sponsor</a>
+          </li>
+          <li class="mob-single-link"><a href="#" class="mob-link">News</a></li>
+          <li class="mob-single-link">
+            <a href="#" class="mob-link">CC Campaign</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <header class="header">
+      <nav class="social-link">
+        <ul class="cont-list-author">
+          <li class="single-cont-author">
+            <a href="#" class="link-nav"><i class="fab fa-facebook-f"></i></a>
+          </li>
+          <li class="single-cont-author">
+            <a href="#" class="link-nav"><i class="fab fa-twitter"></i></a>
+          </li>
+          <li class="single-cont-author">
+            <a href="#" class="link-nav">English</a>
+          </li>
+          <li class="single-cont-author">
+            <a href="#" class="link-nav">Mypage</a>
+          </li>
+          <li class="single-cont-author">
+            <a href="#" class="link-nav">Logout</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="cont-navbar">
+        <div class="navbar-wrapper">
+          <a href="./index.html" class="link-page">
+            <figure class="logo-img-cont">
+              <img
+                src="./images/logo.png"
+                alt="logo"
+                class="logo-footer logo-navbar"
+              />
+              <div class="logo-text">
+                <h1 class="website-name logo-name">Earth</h1>
```

Comment:
- [ ] There should be no more than 1, `h1` tag in each page. _Currently you have **8** in your index.html page and **8** in your about page_